### PR TITLE
Abar to bar integration

### PR DESCRIPTION
**Pull request by harshadptl**
* **make sure that you have executed all the following process and no errors occur**
  - [x] make fmt
  - [x] make lint
  - [x] make test

* **The major changes of this PR**

* **Extra documentations**

### CODE REVIEW

**Review by akhilpeddireddy — APPROVED** [review]

File: src/components/finutils/src/bins/fn.yml [hunk 1]
```diff
@@ -533,6 +533,43 @@ subcommands:
...
+        - randomizer:
+            help: Randomizer for the input Anon BAR
+            short: r
+            long: randomizer
+            takes_value: true
+            value_name: RANDOMIZER
+            required: true
+        - to-xfr-public-key:
+            help: Xfr public key of the receiver
+            short: tp
```
Comment (findora-sun) [inline src/components/finutils/src/bins/fn.yml:555]: Refs [clap short function](https://docs.rs/clap/2.34.0/clap/struct.Arg.html#method.short)

NOTE: Any leading - characters will be stripped, and only the first non - character will be used as the [short](https://docs.rs/clap/2.34.0/clap/struct.Arg.html#method.short) version.

Multiple places `short` use characters, inlcude `feat-triple-masking-v3` branch. Maybe better to fix them together.

Comment (harshadptl) [reply src/components/finutils/src/bins/fn.yml]: Have fixed this

File: src/components/finutils/src/common/mod.rs [hunk 2]
```diff
@@ -808,6 +808,52 @@ pub fn convert_bar2abar(
...
+    to_xfr_public_key: &str,
+    fee_xfr_seckey: Option<&str>,
+    confidential_am: bool,
+    confidential_ty: bool,
+) -> Result<()> {
+    let from = wallet::anon_secret_key_from_base64(axfr_secret_key.as_str())
+        .c(d!("invalid 'from-axfr-secret-key'"))?;
+    let from_secret_key =
+        wallet::x_secret_key_from_base64(dec_key.as_str()).c(d!("invalid dec_key"))?;
+    let to = wallet::public_key_from_bech32(to_xfr_public_key)
```
Comment (findora-sun) [inline src/components/finutils/src/common/mod.rs:825]: Refs `transfer` function, I think when use `xfr-public-key`, it is a `public_key_from_base64`. and use `to-wallet-address` it is a `public_key_from_bech32`.

So, here is `public_key_from_base64` ?

Comment (harshadptl) [reply src/components/finutils/src/common/mod.rs]: I have pushed a fix for this

**Pull request merged**